### PR TITLE
Feature muhammad 1

### PR DESCRIPTION
**Pull request by younus1804**

### CODE REVIEW

File: Pun1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ How does a Penguin build a house? Igloos it together
```
Comment (shalamike) [inline Pun1.txt:1]: nice

**Review by shalamike — APPROVED** [review]

File: Pun1.txt (hunk 1 above)
Comment (Moodhunter34) [reply Pun1.txt:1]: yeah all good

**Review by peter5600 — APPROVED** [review]
Funny

**Review by Moodhunter34 — APPROVED** [review]
all good mate

**Pull request merged**